codesearch/token/token.go

```diff
@@ -351,7 +353,8 @@ func (tt *SparseNgramTokenizer) Type() types.FieldType {
 }
 
 func (tt *SparseNgramTokenizer) Ngram() []byte {
-	return []byte(tt.ngrams[len(tt.ngrams)-1])
+	gram := tt.ngrams[len(tt.ngrams)-1]
+	return unsafe.Slice(unsafe.StringData(gram), len(gram))
```

Comment:
Just confirming that I checked documentation and agree this is all operating on byte counts and not runes

Reply:
ayyyy